### PR TITLE
Documentation update - configuration/introduction.md 

### PR DESCRIPTION
**Pull request by BKeyport**
refer to #211 

This documents the environment variables, and adds examples common to the forums. 

This is fully working as of 12/26/2023, based on discussion in main repository, issue [#3302](https://github.com/MichMich/MagicMirror/issues/3302)

### CODE REVIEW

File: configuration/introduction.md [hunk 1]
```diff
@@ -93,15 +165,110 @@ MY_HTTPS=false
...
+cd ~/MagicMirror
+npm start
 ```
 
-#### Real world example of `config.js.template`
+Starting Script 2 (mm2.sh): 
+
+```bash
+cd ~/MagicMirror
+export MM_CONFIG_FILE=${pwd}/config/config2.js
```
Comment (khassel) [inline configuration/introduction.md:218]: can you please omit the `${pwd}/` part here?

Comment (BKeyport) [reply configuration/introduction.md]: Fixed.

File: configuration/introduction.md [hunk 2]
```diff
@@ -1,47 +1,119 @@
...
-
-#### `config.js` and `config.js.template`
-
-You can provide a `config.js.template` instead of a `config.js` file. The
-difference is, that you can use environment variables in the
-`config.js.template`. When starting MagicMirror² a `config.js` is created from
-`config.js.template` and the variables are resolved.
+
+
+### Extremely Basic Configuration of MagicMirror²:
```
Comment (KristjanESPERANTO) [inline configuration/introduction.md:5]: I suggest to change this heading to "Minimum Configuration of MagicMirror²".

File: configuration/introduction.md [hunk 3]
```diff
@@ -1,47 +1,119 @@
...
+           config: {
+       	}
+       },
+     ],
+   };
+   ```
+
+   See [module configuration](/modules/configuration.md) for more information and examples. 
+
+4.  run your magic mirror.
```
Comment (KristjanESPERANTO) [inline configuration/introduction.md:28]: Would a note on how to run it be useful here?

Comment (BKeyport) [reply configuration/introduction.md:28]: There's too many ways to run it, and it's covered previously in the documentation, so I didn't include it here.

File: configuration/introduction.md (hunk 2 above)
Comment (BKeyport) [reply configuration/introduction.md:5]: of course, it's not the minimum. I decided to go with the most basic 'useful' configuration. In playing with the mirror for documentation, you CAN get away with 
```
let config = { }
```

It'll run, it won't be happy, but it'll run.

File: configuration/introduction.md (hunk 3 above)
Comment (KristjanESPERANTO) [reply configuration/introduction.md:28]: Ah, okay. Perhaps a link or reference where it is described?

File: configuration/introduction.md (hunk 2 above)
Comment (KristjanESPERANTO) [reply configuration/introduction.md:5]: You are right. Somehow the word "Extremely" just bothers me. But maybe I just have a strange feeling for the language

File: configuration/introduction.md (hunk 3 above)
Comment (BKeyport) [reply configuration/introduction.md]: ooh, good call. Adjusted.

File: configuration/introduction.md (hunk 2 above)
Comment (BKeyport) [reply configuration/introduction.md:5]: Thinking on it, you're right. I'll strike that.

Comment (KristjanESPERANTO) [reply configuration/introduction.md]: Nice! Thanks.

File: configuration/introduction.md (hunk 3 above)
Comment (KristjanESPERANTO) [reply configuration/introduction.md]: Nice! Thanks :slightly_smiling_face:

**Comment by rejas** [comment]
Just wondering if this needs to be against develop or master really? But anyay, good work, dont have to hurry to fix it since the docs repo is released more frequently in case of master work

**Comment by BKeyport** [comment]
My fault, I didn't realize there was a develop branch.

**Comment by khassel** [comment]
@rejas can we merge this now?

**Comment by BKeyport** [comment]
Due to errors, I had to remove this PR. My fork blew up and corrupted itself.

**Comment by KristjanESPERANTO** [comment]
Too bad. I hope your changes haven't been lost.

**Comment by BKeyport** [comment]
No, I keep backups of everything.. It'll be back up in a new pull in a little bit.

**Comment by BKeyport** [comment]
Refer to #220 for new pull request.

**Pull request closed without merging**